docs/getstarted/themes.md

```diff
@@ -47,10 +47,10 @@ Windows and macOS support light and dark color schemes. There is a setting, `win
 
 To customize the themes that are used when a color scheme changes, you can set the preferred light, dark, and high contrast themes with the settings:
 
-* `workbench.preferredLightColorTheme` - defaults to "Default Light+"
-* `workbench.preferredDarkColorTheme` - defaults to "Default Dark+"
-* `workbench.preferredHighContrastColorTheme` - defaults to "Default High Contrast"
-* `workbench.preferredHighContrastLightColorTheme` - defaults to "Default High Contrast Light"
+* `workbench.preferredLightColorTheme` - defaults to "Light Modern"
```

Comment:
The setting values still include "Default" in the string.

Reply:
Yeah, I see that. This is the confusion that inspired me to make the PR. The documentation is meant to use the underlying names, but as a user, I use the names in the UI, and they're different.

So thanks for your patience. I'll revise (again). What do you think about noting in the document that the names are different in the UI?"

Reply:
This section could be changed to use the Settings editor names instead of the settings IDs and in that case, using the Settings editor visible name would be correct.

To customize the themes that are used when a color scheme changes, you can set the preferred light, dark, and high contrast themes in the Settings editor:

* Workbench: Preferred Dark Color Theme - defaults to "Dark Modern"
* Workbench: Preferred High Contrast Color Theme - defaults to "Dark High Contrast"
* etc

Reply:
Nice to call out the Settings editor here as the previous section used the "Select Color Theme" picker UI